diagrams/__init__.py

```diff
@@ -392,4 +394,111 @@ def _load_icon(self):
         return os.path.join(basedir.parent, self._icon_dir, self._icon)
 
 
+class Edge:
```

Comment:
It should have default edge attributes. All nodes use `Sans-Serif` as default, edge also should use it as default.

Suggested default option is here:

```python
_default_edge_attrs = {
    "fontcolor": "#2D3436",
    "fontname": "Sans-Serif",
    "fontsize": "13",
}
```

Reply:
It will come into conflict with:
```
    _default_graph_attrs = {
        "pad": "2.0",
        "splines": "ortho",
        "nodesep": "0.60",
        "ranksep": "0.75",
        "fontname": "Sans-Serif",
        "fontsize": "15",
        "fontcolor": "#2D3436",
``` 
in case you defined it like:
```
with Diagram(name="Advanced Web Service with On-Premise (colored)", "fontname": "Arial"):
    Node("n") - Edge("label") - Node("nn")
```

Edge will have Sans-Serif font instead of Arial, which looks inconsistent.

Reply:
But it already looks inconsistent. Graph has `Sans-Serif` (diagrams default), but Edge has `Times-Roman` as default (graphviz default).

So I want to keep the font consistently at least for (diagrams) default font (`Sans-Serif`).

As you might have seen, the Graph and Node already define default font as `Sans-Serif` (so it already have conflict problem you said. But it is not a big deal, and it is intended):
 https://github.com/mingrammer/diagrams/pull/48/files#diff-5bb427ec68a1e4f4485ddf77d49b90a8R46-R71

Reply:
Oh, got it. Fixed.